src/sentry/ingest/transaction_clusterer/datasource/redis.py

```diff
@@ -191,20 +191,14 @@ def record_span_descriptions(
             continue
         safe_execute(_record_sample, ClustererNamespace.SPANS, project, description)
 
-        update_rule_rate = options.get("span_descs.bump-lifetime-sample-rate")
-        if update_rule_rate and random.random() < update_rule_rate:
-            safe_execute(
-                _update_span_description_rule_lifetime, project, event_data, _with_transaction=False
-            )
```

Comment:
Relay does no longer use these rules, so no need to bump their lifetimes.

Reply:
Is there any chance old relays are using these rules? Or was this feature always behind a flag?

Reply:
Good question! I excluded them from the project config in https://github.com/getsentry/sentry/pull/58689, so no, not anymore.